static/js/components/forms/CheckoutForm.js

```diff
@@ -182,6 +182,17 @@ export class InnerCheckoutForm extends React.Component<InnerProps, InnerState> {
     })
   }
 
+  isPromoCodeApplied = () => {
+    const { coupon, errors, values } = this.props
+
+    return (
+      !errors.coupons &&
+      values.couponCode &&
+      this.props.coupon &&
```

Comment:
```suggestion
      coupon &&
```

Reply:
sorry, my bad. I supposed to fix it but somehow it got skipped.